lib/rules/sotd/implementation.js

```diff
@@ -23,6 +23,9 @@ exports.check = function (sr, done) {
 
           for (item in url_candidates) {
             sr.info(self, 'candidate', {text: url_candidates[item], url: item});
+            if (!item.match(/^https:\/\//)) {
+              sr.error(self, 'link-should-be-https', { link: item });
```

Comment:
You probably don't want to add the error in the `for` loop. If we have find 10 links, we only want 1 error.